.changeset/curvy-penguins-act.md

```diff
@@ -0,0 +1,7 @@
+---
+'@astrojs/tailwind': major
+---
+
+Deprecates the integration
+
+This release doesn't change how the integration behaves. However, it deprecates it since Tailwind CSS now offers a Vite plugin which is the preferred way to use Tailwind 4 in Astro. If you want to migrate, uninstall `@astrojs/tailwind` and follow the [Tailwind documentation for manual installation](https://tailwindcss.com/docs/installation/framework-guides/astro).
```

Comment:
``````suggestion
Tailwind CSS now offers a Vite plugin which is the preferred way to use Tailwind 4 in Astro. Please uninstall `@astrojs/tailwind` and follow the [Tailwind documentation for manual installation](https://tailwindcss.com/docs/installation/framework-guides/astro).

This updated major version is only provided as a convenience for existing projects until they are able to migrate to the new plugin. It offers no additional functionality and is no longer recommended, but may continue to be used in your projects until it is removed entirely.
```
Maybe something like this?